modules/vpn/main.tf

```diff
@@ -107,13 +107,13 @@ resource "aws_iam_access_key" "furyagent" {
 }
 
 resource "aws_iam_policy_attachment" "furyagent" {
-  name       = "${var.name}-vpn"
+  name       = "${var.name}-${var.vpn_bucket_name_prefix}"
```

Comment:
I would leave vpn here, and deal with it in the migration procedure if necessary.